src/models/entry/index.ts

```diff
@@ -0,0 +1,24 @@
+interface Entry {
+  id: string;
+  data: EntryData;
+}
+
+export interface EntryData {
+  name: string;
+  latlng: {
+    latitude: number;
+    longitude: number;
+  };
+}
+
+export const defaultEntry = {
+  name: "Default Entry",
+  latlng: {
+    latitude: 0,
+    longitude: 0,
+  },
+};
```

Comment:
suggestion: as suggested in each reference, this can be removed

Reply:
done